res/controllers/Denon-MC7000-scripts.js

```diff
@@ -890,17 +1037,50 @@ MC7000.censor = function(channel, control, value, status, group) {
     }
 };
 
+// Param Button for Pitch Play to increase or decrease pitch, Star rating otherwise
+MC7000.StarsDown = function(channel, control, value, status, group) {
+    var deckNumber = script.deckFromGroup(group);
+    var deckIndex = deckNumber - 1;
+    if (value === 0x00) {
+        //return; // don't respond to note off messages
```

Comment:
Is this `return;` needed or not? If not remove the whole if branch.

Reply:
done